### DISCUSSION TITLE
Rescue compatibility with Nanopore data

### DISCUSSION THREAD

**Discussion by MartinezRuiz-Carlos** [ConesaLab/SQANTI3#292] [Q&A] [answered]
Hi all, I have been running SQANTI3 on Nanopore long read data (cDNA), so far so good. I know the tool is meant to be compatible with Nanopore input, but there are a couple points that I think need clarification.
- First one, probably simplest, I notice that the in the GTF output from SQANTI3 QC, the second column gets automatically named PacBio for every entry. I imagine this is just aesthetic, but I just wanted to double check. I run the sqant3_qc.py with the `--force_id_ignore` option enabled, as recommended for non-PacBio data.
- The second one might be a bit more involved. From the paper, the rescue part of the pipeline runs minimap2 with the map-hifi option. But isn't this option specifically only for CCS reads from Pac-Bio? I am aware no reads are actually being mapped, but would this be an issue for non-PacBio derived annotations?
Many thanks!

**Comment by aarzalluz** [comment, marked as answer]
Hi,
 
> * First one, probably simplest, I notice that the in the GTF output from SQANTI3 QC, the second column gets automatically named PacBio for every entry. I imagine this is just aesthetic, but I just wanted to double check. I run the sqant3_qc.py with the `--force_id_ignore` option enabled, as recommended for non-PacBio data.

The naming is in fact irrelevant to run SQANTI3 Rescue. There's no need to worry!

> * The second one might be a bit more involved. From the paper, the rescue part of the pipeline runs minimap2 with the map-hifi option. But isn't this option specifically only for CCS reads from Pac-Bio? I am aware no reads are actually being mapped, but would this be an issue for non-PacBio derived annotations?

Since we are mapping transcript models instead of reads, we believed `map-hifi` to be the closest parameter to our real-life scenario, as HiFi reads are expected to have higher mapping accuracy than regular ONT reads, for example. While we did not test the remaining parameters, we do allow up to 6 supplementary alignments to be output, and we have observed that both highly similar and more divergent artifacts are assigned to same-gene rescue targets via mapping. Our downstream filters are the way to ensure that we are not introducing noise in the transcriptome.

Since your ONT transcripts will be pre-processed by a long-read transcriptome reconstruction algorithm, I don't think you need to worry about this either.

Best,

Ángeles
